kratos/processes/apply_periodic_boundary_condition_process.cpp

```diff
@@ -153,7 +153,7 @@ void ApplyPeriodicConditionProcess::ApplyConstraintsForPeriodicConditions()
 
     IndexType num_slaves_found = 0;
 
-    block_for_each(mrSlaveModelPart.Nodes(), num_slaves_found, [&](Node<3>& rNode, IndexType& r_num_slaves_found){
+    num_slaves_found = block_for_each<SumReduction<IndexType>>(mrSlaveModelPart.Nodes(), num_slaves_found, [&](Node<3>& rNode, IndexType& r_num_slaves_found){
```

Comment:
Is it okay now? @philbucher

Reply:
`num_slaves_found` should now be declared inside, you don't need the TLS
This should simplify it a bit

Reply:
I didn't get you . You mean _**r_num_slaves_found**_ show be declare inside and remove it from TLS?
![image](https://user-images.githubusercontent.com/46954821/108543297-2d6b0e00-72e5-11eb-9417-085c1a6a67f9.png)

Reply:
yes exactly